website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -52,165 +52,8 @@ unit_tests:
 
```

Comment:
@matthewshaver we chatted about adding a heading to the file above. Something like 
`## File structure`

@graciegoheen would this make sense for all files  that lay out the configs or properties and their option like we do above?